src/core/AnimatedValue.js

```diff
@@ -14,21 +16,44 @@ export default class AnimatedValue extends AnimatedNode {
   }
 
   __detach() {
+    this.__detachAnimation();
     super.__detach();
   }
 
+  __detachAnimation(result) {
```

Comment:
Looks like we no longer call `detachAnimation` when the animation finishes

Reply:
I can't get it

Reply:
We call by ` value.__setAnimation(null, !wasStopped);`